sci-electronics/librepcb/librepcb-1.1.0.ebuild

```diff
@@ -0,0 +1,75 @@
+# Copyright 2009-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake virtualx xdg
+
+DESCRIPTION="Free EDA software to develop printed circuit boards"
+HOMEPAGE="https://librepcb.org/"
+SRC_URI="https://download.librepcb.org/releases/${PV}/${P}-source.zip"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64"
+
+IUSE="opencascade +qt5 qt6"
+REQUIRED_USE="|| ( qt5 qt6 )"
```

Comment:
https://wiki.gentoo.org/wiki/Project:Qt/Policies#Requires_one_of_two_Qt_versions

qt5 and qt6 cant be on at the same time so the qt5 use flag does nothing other than pull in the dependencies

Reply:
But what I mean is https://wiki.gentoo.org/wiki/Project:Qt/Policies#Requires_at_least_one_Qt_version

Reply:
But this ebuild doesn't do that. If you enable qt6 you dont get qt5. If you wanted to that you would have to add it with multibuild, but even so I dont see why you would do that.

> -DQT_MAJOR_VERSION=$(usex qt6 6 5)

Reply:
After chat on #gentoo-qt I realize you're right.